### PR TITLE
remove usage of OLD_LLVMPASS

### PR DESCRIPTION
**Pull request by catenacyber**
Following https://github.com/google/oss-fuzz/pull/12365
cc @maflcko

### CODE REVIEW

File: projects/bearssl/Dockerfile [hunk 1]
```diff
@@ -24,4 +24,3 @@ RUN wget https://archives.boost.io/release/1.84.0/source/boost_1_84_0.tar.bz2
 COPY build.sh $SRC/
 # This is to fix Fuzz Introspector build by using LLVM old pass manager
```
[the remark this answers is not in the document]
Comment (catenacyber) [reply projects/bearssl/Dockerfile]: done in fixup commit :-)

**Comment by oliverchang** [comment]
thanks!

**Comment by oliverchang** [comment]
/gcbrun trial_build.py all --fuzzing-engines libfuzzer --sanitizers introspector

**Comment by catenacyber** [comment]
What is gcbrun result ?

**Pull request merged**